docs/notifications.json

```diff
@@ -18,5 +18,10 @@
     "id": 76,
     "title": "<b>Unveiling Charts: Alpha release is live</b>",
     "text": "We're starting with bars, lines, and scatter charts. <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-introduce-charts\" href=\"https://mui.com/x/react-charts/\">Try X Charts now</a>, and let us know what you need."
+  },
+  {
+    "id": 77,
+    "title": "<b>MUI Toolpad is now in Beta</b>",
+    "text": "Import in your codebase, connect to datasources and build UI in minutes! Try <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-toolpad-introduce-beta\" href=\"https://mui.com/toolpad/\"> Toolpad today</a>, and share your feedback."
```

Comment:
Why not link the blog post?

Reply:
Since we linked blog directly on the hero section of homepage, I thought we can skip it. But I agree makes sense for the users to read the blog first.

Reply:
I realized the same and made the changes.